tests/tests_suite_test.go

```diff
@@ -279,9 +280,18 @@ func (s *existingSspStrategy) Init() {
 }
 
 func (s *existingSspStrategy) Cleanup() {
-	if s.ssp != nil {
-		s.RevertToOriginalSspCr()
+	if s.ssp == nil {
+		return
 	}
+
+	s.RevertToOriginalSspCr()
+
+	// Waiting for PVCs that are imported by DataImportCrons to be in bound state.
+	//
+	// The tests in dataSources_test.go remove and recreate DataImportCrons, and
+	// they can cause the PVCs to be removed. To keep the cluster in healthy state after tests,
+	// we wait until PVCs are reimported.
+	s.waitUntilPVCsAreBound()
```

Comment:
Is it also needed initially before running the first test?

Reply:
No, we assume that the PVCs are bound. In the tier1 test pipeline, the health check is run between tests of components, so we can assume that it run before SSP tests.